pkg/controller/actions/gc/action_gc.go

```diff
@@ -79,6 +81,12 @@ func WithTypePredicate(value TypePredicateFn) ActionOpts {
 	}
 }
 
+func WithCollectOwned(value bool) ActionOpts {
```

Comment:
```suggestion
func WithOnlyCollectOwned(value bool) ActionOpts {
```

I think the "only" part should be in here somewhere. Otherwise it might kinda make it sound like it collects non-owned by default and not owned, and it has to be explicitly told to collect owned. Of course that's nonsense, but I think having "only" be part of the name will make it more immediately obvious. WDYT?

Reply:
fixed